hibike/hibikeDevices.json

```diff
@@ -28,13 +28,14 @@
     },
 
     {"id": 4,     "name": "BatteryBuzzer", "params": [    
-                                                          {"number": 0  , "name": "connected"  , "type": "bool"    , "read": true , "write": false },
-                                                          {"number": 1  , "name": "safe"       , "type": "bool"    , "read": true , "write": false },
-                                                          {"number": 2  , "name": "cell1"      , "type": "float"   , "read": true , "write": false },
-                                                          {"number": 3  , "name": "cell2"      , "type": "float"   , "read": true , "write": false },
-                                                          {"number": 4  , "name": "cell3"      , "type": "float"   , "read": true , "write": false },
-                                                          {"number": 5  , "name": "total"      , "type": "float"   , "read": true , "write": false },
-                                                          {"number": 6  , "name": "callibrate" , "type": "bool"    , "read": false, "write": true  }
+                                                          {"number": 0  , "name": "unsafe_status"  , "type": "bool"    , "read": true , "write": false },
```

Comment:
To be honest, the name `unsafe_status` is confusing to me, because I'm not sure whether true means safe, or unsafe. The old name "safe" was clearer (and something like `unsafe`, `is_unsafe`, etc. would also be clearer)